nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -572,6 +572,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The `mpich` package expression now requires `withPm` to be a list, e.g. `"hydra:gforker"` becomes `[ "hydra" "gforker" ]`.
 
+- To facilitate dependency injection, the `imgui` package now builds a static
+  archive using vcpkg' CMake rules. It now installs "impl" headers selectively
+  instead of by a wildcard. Use `imgui.src` to if you just want to access the
+  unpacked sources.
```

Comment:
```suggestion
- To facilitate dependency injection, the `imgui` package now builds a static archive using vcpkg' CMake rules.
  It now installs "impl" headers selectively instead of by a wildcard. Use `imgui.src` to if you just want to access the unpacked sources.
```
easier to read and we don't have an 80 line length limit in nixpkgs anyway